src/state/atoms.ts

```diff
@@ -0,0 +1,4 @@
+import { atom } from 'jotai';
+// setup initial chrome atoms
+export const contextSwitcherOpenAtom = atom(false);
+export const activeModuleAtom = atom<string | undefined>(undefined);
```

Comment:
@Hyperkid123 curious on your thoughts on utilizing a pattern where we export a named type in support of atoms versus inline primitive types. IE:
```ts
// src/state/atom-types.ts or similar
export type ActiveModule = string | undefined;

// src/state/atoms.ts
export const activeModuleAtom = atom<ActiveModule>(undefined);
```

I get for this particular atom it's not likely it'll change drastically - but for those that might it would be a good habit to get in?

Reply:
And it may be beneficial to reserve `atoms.ts` as the main aggregate file that simply exports from more specific files in the directory - for example `context-switcher-atoms.ts` (to hopefully give us our best chance of avoiding merge conflicts in the main `atoms.ts`).

Reply:
I think splitting atoms into individual files is something we want to do. Don't know about detaching the types to be in separate files. It seems like too much of a separation especially when this chrome is not a library to be consumed by anyone else.